src/analysis/normed/group/add_circle.lean

```diff
@@ -211,6 +218,43 @@ begin
         linarith [abs_eq_neg_self.mpr hp.le] } } },
 end
 
+section finite_order_points
+
+variables {p} [hp : fact (0 < p)]
+include hp
+
+lemma norm_div_nat_cast {m n : ℕ} :
+  ∥(↑((↑m / ↑n) * p) : add_circle p)∥ = p * (↑(min (m % n) (n - m % n)) / n) :=
```

Comment:
I think you'll need to update the norm symbols to make this pass CI

Reply:
Thanks, done.